engine/runtime/src/main/java/org/enso/interpreter/runtime/data/hash/EnsoHashMap.java

```diff
@@ -98,7 +98,7 @@ boolean hasHashEntries() {
   }
 
   @ExportMessage
-  int getHashSize() {
+  public int getHashSize() {
```

Comment:
Why do we need this method `public`. It intentionally wasn't.

Reply:
Reverted to package protected. See my comment at https://github.com/enso-org/enso/pull/10555#discussion_r1705253874